OneTimeKit/Services/OTQRCreatorService.m

```diff
@@ -0,0 +1,63 @@
+#import <Foundation/Foundation.h>
+#import <CoreImage/CoreImage.h>
+#import <UIKit/UIKit.h>
+
+@interface OTQRCreatorService : NSObject
+
++ (UIImage *)generateQRCodeFromString:(NSString *)string;
+
+@end
+ 
+@implementation OTQRCreatorService
+
++ (CIImage *)generateQRCodeImageFromString:(NSString *)string {
```

Comment:
This is implemented in https://github.com/leptos-null/OneTime/blob/88395900c67852bb9e7597c2bdae5a2a150b1844/OneTimeKit/Services/OTQRService.m#L56